src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -57,6 +63,89 @@ def _build_x509_name(backend, x509_name):
     return x509.Name(attributes)
 
 
+def _build_general_name(backend, gn):
+    # dNSName
+    if gn.type == 2:
```

Comment:
Good question... I don't know where those constants would live in OpenSSL. It seems likely that they do exist though...